### PR TITLE
Add migrations

### PR DESCRIPTION
**Pull request by kdenkiewicz**

### CODE REVIEW

File: migrations/env.py [hunk 1]
```diff
@@ -0,0 +1,96 @@
+from __future__ import with_statement
```
Comment (msoltysik) [inline migrations/env.py]: unused?

File: migrations/versions/26f12d3a60bd_.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
+"""empty message
```
Comment (msoltysik) [inline migrations/versions/26f12d3a60bd_.py]: Let's name it.

File: migrations/versions/26f12d3a60bd_.py [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('questions', sa.Column('subject', sa.String(), server_default='', nullable=False))
+    op.alter_column('questions', 'question',
+               existing_type=sa.VARCHAR(),
+               nullable=False)
+    # ### end Alembic commands ###
+
+
+def downgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
```
Comment (msoltysik) [inline migrations/versions/26f12d3a60bd_.py]: if adjusted, just remove the comments.

File: migrations/versions/9caea0f07721_.py [hunk 4]
```diff
@@ -0,0 +1,28 @@
+"""empty message
```
Comment (msoltysik) [inline migrations/versions/9caea0f07721_.py]: Let's name it.

File: migrations/versions/a060c3261b67_.py [hunk 5]
```diff
@@ -0,0 +1,36 @@
+"""empty message
```
Comment (msoltysik) [inline migrations/versions/a060c3261b67_.py]: Let's name it.

File: migrations/versions/a060c3261b67_.py [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+# revision identifiers, used by Alembic.
+revision = 'a060c3261b67'
+down_revision = '26f12d3a60bd'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
```
Comment (msoltysik) [inline migrations/versions/a060c3261b67_.py]: not needed? 🌷

**Review by msoltysik — APPROVED** [review]

**Review by marcin992 — APPROVED** [review]
Good job! :D

File: migrations/env.py (hunk 1 above)
Comment (kdenkiewicz) [reply migrations/env.py]: it was generated automatically, I'll remove it

Comment (chevrette) [reply migrations/env.py]: it was generated automatically, I'll remove it

**Pull request merged**